test_runner/performance/test_bulk_tenant_create.py

```diff
@@ -0,0 +1,58 @@
+import timeit
+import pytest
+
+from fixtures.zenith_fixtures import (
+    TenantFactory,
+    ZenithCli,
+    PostgresFactory,
+)
+
+pytest_plugins = ("fixtures.benchmark_fixture")
+
+# Run bulk tenant creation test.
+#
+# Collects metrics:
+#
+# 1. Time to create {1,10,50} tenants
+# 2. Average creation time per tenant
+
+
+@pytest.mark.parametrize('tenants_count', [1, 5, 10])
+@pytest.mark.parametrize('use_wal_acceptors', ['with_wa', 'without_wa'])
```

Comment:
nit: could be just `[True, False]` and corresponding check could be `if use_wal_acceptors:`

Reply:
@kelvich 
Yes, it was initially (but I changed to `str` to have more readable output - `with_wa-10` instead of `True-10`). I can easy change to boolean if need.

Reply:
ah, makes sense then